Triangulation_3/include/CGAL/Delaunay_triangulation_3.h

```diff
@@ -571,11 +571,6 @@ class Delaunay_triangulation_3<Gt, Tds_, Default, Lock_data_structure_>
                                           OutputItCells fit);
 
 public:
-
-#ifndef CGAL_NO_DEPRECATED_CODE
-  CGAL_DEPRECATED Vertex_handle move_point(Vertex_handle v, const Point & p);
-#endif
```

Comment:
That must be announce in the `CHANGES.md` as a breaking change.

Reply:
Yes I know, I was writing about that...

I was wondering if it makes sense. It was removed from the documentation when it was deprecated in 3.7 (https://doc.cgal.org/Manual/3.7/doc_html/cgal_manual/Triangulation_3_ref/Class_Delaunay_triangulation_3.html#Cross_link_anchor_973) and its depreciation wasn't even documented in change-log.

The similar functions in T3, RT3 etc are not even documented.